docs/integrations/rest-api.mdx

```diff
@@ -0,0 +1,145 @@
+---
+sidebar_position: 3
+title: REST API
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Using the REST API to run flows
+
+In this guide we'll demonstrate how you can use the REST API to integrate your flow into existing systems. All API 
+calls will be exemplified using the command line tool cURL, but any tool you're using that has support for making 
+arbitrary REST API calls will work similarly.
+
+<br/>
+
+## Prerequisites
+
+You need to have a [Cradl AI](https://app.cradl.ai/) account and have created a flow. If you haven't, go to 
+[flows](https://app.cradl.ai/flows) and create one.
+
+<br/>
+
+## Step 1: Acquire API credentials
+
+Go to the [API tab](https://app.cradl.ai/appclients) in Cradl and use existing credentials or create new ones. Read 
+more on how to use the credentials to make authenticated requests [here](../reference/authentication.md)
+
+<br/>
+
+## Step 2: Run a flow with a document
+
+Enable the REST API input integration on your flow by going to the flow builder in Cradl AI and locate the `workflowId` 
+that will be required to run a flow.
+
+![REST API Input Integration](/img/integrations/rest-api/rest-api-input.png)
+
+<br/>
+
+### Uploading a document
+
+To begin, we need to upload the document that you want to send to the flow. In order to do that we need to create a 
+document handle in the API.
+
+```bash
+curl -X POST https://api.lucidtech.ai/v1/documents 
+```
+
+:::caution
+
+Remember to add the Authorization header to all API requests demonstrated in this guide or the request will fail. Read
+more about how to acquire and add the Authorization header 
+[here](../reference/authentication.md#getting-an-access-token-manually)
+
+:::
```

Comment:
Maybe move this above the first call to make sure that the reader has already seen it if everything fails.

Reply:
I moved the caution to step 1, what do you think of that placement?